.github/workflows/deploy.yml

```diff
@@ -1,49 +1,52 @@
-# This is a basic workflow to help you get started with Actions
-
 name: CI
 
-# Controls when the workflow will run
 on:
-  # Triggers the workflow on push or pull request events but only for the dev branch
   push:
     branches: [ dev ]
   pull_request:
     branches: [ dev ]
 
-  # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
 
-# A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
-  # This workflow contains a single job called "build"
   build:
-    # The type of runner that the job will run on
     runs-on: ubuntu-latest
-
-    # Steps represent a sequence of tasks that will be executed as part of the job
-    # .github/workflows/app.yaml
     steps:
-      - name: Check out repository code
-        uses: actions/checkout@v2
-
-      # Setup Python (faster than using Python container)
-      - name: Setup Python
-        uses: actions/setup-python@v2
-        with:
-          python-version: "3.x"
-
-      - name: download poetry
+      - uses: actions/checkout@v2
+      - name: Create File app/.env
         run: |
-          curl https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py > get-poetry
-      - id: Install_poetry
-        run: python get-poetry
-
-      - name: activate poetry
+          echo SECRET_KEY='146EtHWvxaInR3YB3d2Cdtn4emQcUBWw' > app/.env
```

Comment:
this is fine for now. generally it's bad to store the passwords and other secrets plain text in the source code. you'll need to use some sort of secret storage/encryption

Reply:
https://www.theserverside.com/blog/Coffee-Talk-Java-News-Stories-and-Opinions/GitHub-Actions-Secrets-Example-Token-Tutorial as an option something like this. but again, this is fine for now.

Reply:
done(test values)